### PR TITLE
CoqIDE: Make tabs reorderable and fix preference on tabs position

### PR DESCRIPTION
**Pull request by Frigory33**
- [x] Added **changelog**.

### CODE REVIEW

**Review by herbelin — APPROVED** [review]
Moving document tabs now works for me.

BTW, what was requiring a restart exactly?

**Comment by Frigory33** [comment]
> BTW, what was requiring a restart exactly?

Edit > Preferences > Misc > Vertical tabs and Tabs on opposite side. (In my branch coqide-preferences, I have merged these two settings in a single one.)

**Comment by herbelin** [comment]
> Edit > Preferences > Misc > Vertical tabs and Tabs on opposite side. (In my branch coqide-preferences, I have merged these two settings in a single one.)

OK, found, nice.

**Comment by SkySkimmer** [comment]
@coq/coqide-maintainers please assign

**Comment by herbelin** [comment]
OK, going to merge if no objection.

**Comment by Frigory33** [comment]
> OK, going to merge if no objection.

Thanks. Note that @jfehrle feared that moving the tabs would make things buggy, but I think I did what was needed to make it work.

**Comment by jfehrle** [comment]
@herbelin I said I'd review it when I got back from vacation.  I'm now back and should get to it within a few days.  Is this time critical?

**Comment by herbelin** [comment]
> Is this time critical?

No. This will be anyway for 8.21.

**Comment by herbelin** [comment]
Also unassigning so that you can assign.

File: doc/changelog/10-coqide/19166-coqide.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+- **Fixed:**
+  Changing document tabs position no longer needs a restart
```
Comment (jfehrle) [inline doc/changelog/10-coqide/19166-coqide.rst:2]:
```suggestion
  Changing the position of buffer names (vertical or horizontal) no longer needs a restart
```

**Review by jfehrle — APPROVED** [review]
Looks good.  Would you tweak the changelog for clarity, then I will merge?  Thanks.

**Comment by Frigory33** [comment]
It is not only vertical or horizontal, also bottom or right with the other checkbox.

Tweaking done. Thank you for merging.

**Comment by jfehrle** [comment]
@coqbot: merge now

@Frigory33: thanks for the PR!

**Pull request merged**